pkg/backup/backup_compaction.go

```diff
@@ -75,10 +75,18 @@ func maybeStartCompactionJob(
 	triggerJob jobspb.BackupDetails,
 ) (jobspb.JobID, error) {
 	threshold := backupCompactionThreshold.Get(&execCfg.Settings.SV)
-	if threshold == 0 || triggerJob.RevisionHistory ||
-		triggerJob.StartTime.IsEmpty() || triggerJob.ScheduleID == 0 {
+	if threshold == 0 || triggerJob.StartTime.IsEmpty() {
 		return 0, nil
 	}
+
```

Comment:
could you also disallow the following here:
- backups of tenants
- separate incremental location